openslides/core/static/templates/index.html

```diff
@@ -9,6 +9,7 @@
 <link rel="stylesheet" href="static/css/openslides-libs.css">
 <link rel="stylesheet" href="static/css/app.css">
 <script src="static/js/openslides-libs.js"></script>
+<script src="static/ckeditor/ckeditor.js"></script>
```

Comment:
At the moment I use an extra gulp task to copy the files, see gulpfile.js above. I think this would be enough. Integrating only one file more to the openslides-libs.js does not bring salvation to us.